### ISSUE TITLE
PEP257 not being run

### ISSUE THREAD

**Issue by dan-c-underwood** [AtomLinter/linter-flake8#72]
No PEP257 errors are being reported despite `flake8-pep257` or `flake8-docstrings` installed (tested with both).

Running `flake8` in the command line on an example Python file gives the output:

```
build.py:1:1: D100 Missing docstring in public module
build.py:8:1: D103 Missing docstring in public function
build.py:21:1: D103 Missing docstring in public function
```

However these are not displayed by this linter. In the package settings I've tried changing it to recognise `E, W, D` rather than just  `E, W`, but this has no effect.

**Comment by Arcanemagus** [comment]
You know, I was wondering why I wasn't seeing any errors related to that...

Turns out this is a bug in the regex that filters out the input (because `flake8` can't provide nice json output...). This happens _after_ the output of the program, hence why the setting has no effect. I'll throw up a PR that fixes this shortly.

**Comment by cemsbr** [comment]
I'm having this issue.

**Comment by Arcanemagus** [comment]
As it looks like you already discovered, this isn't actually the issue you are experiencing @cemsbr.
